index.js

```diff
@@ -7,14 +23,258 @@ function ask(questionText) {
   });
 }
 
-start();
+//define player object literal
+
+let Player = {
+  name: "",
+  mood: "",
+  inventory: ["first aid kit", "nalgene", "peanut butter M&Ms (melted)"]
+}
+
+//creating class template for state objects (Victim)
+
+class Victim {
+  constructor(name, description, assessIt, interactIt, savedIt, saveNext) {
+    this.name = name
+    this.description = description
+    this.assessIt = assessIt || "There's nothing to assess..."
+    this.interactIt = interactIt || "There's nothing to interact with..."
+    this.savedIt = savedIt
+    this.saveNext = saveNext;
+  }
+
+  //assess the victim method that changes the savedIt flag to true
+  assess() {
+    if (this.assessIt) {
+      console.log(this.assessIt)
+      this.savedIt = true
+    }
+  }
+
+  //interact with the victim method
+  interact() {
+    if (this.interactIt) {
+      console.log(this.interactIt)
+      }
+    }
+  
+
+  
+  //state machine method
+  move(nextVictim) {
```

Comment:
it would be helpful to see more line by line comments. this is a great example of a place that would benefit. another would be a breakdown of your methods.